examples/existing-kafka.yaml

```diff
@@ -0,0 +1,14 @@
+cp-zookeeper:
+  enabled: false
+cp-kafka:
+  enabled: false
+
+humio-core:
+  enabled: true
+  replicas: 3
+  waitFor:
```

Comment:
I'm wondering if we should set `waitFor.kafka.enabled: false` or ignore this setting altogether if `external. kafkaBrokers` is set?

Reply:
Let's just skip the initContainer entirely if `external.kafkaBrokers` is set.